packages/api-aco/src/subscriptions/aftePageCreate.ts

```diff
@@ -0,0 +1,28 @@
+import { ContextPlugin } from "@webiny/api";
+import WebinyError from "@webiny/error";
+import get from "lodash.get";
+
+import { ACOContext } from "~/types";
+
+export const afterPageCreate = () => {
+    return new ContextPlugin<ACOContext>(async context => {
+        context.pageBuilder.onPageAfterCreate.subscribe(async ({ page, meta }) => {
+            try {
+                const folderId = get(meta, "location.folderId", "ROOT");
+                await context.folders.createLink({ id: page.pid, folderId });
+            } catch (error) {
+                /**
+                 * In case the link already exists for the current page, just return.
+                 */
+                if (error.code === "LINK_EXISTS") {
+                    return;
+                }
+
+                throw WebinyError.from(error, {
+                    message: "Error while creating page link.",
+                    code: "AFTER_PAGE_CREATE"
```

Comment:
Codes should include some prefix that makes them easier to find, like `ACO_AFTER_PAGE_CREATE`, then it's clear the error is coming from ACO package.

Reply:
👍